src/controllers/controllermappinginfoenumerator.cpp

```diff
@@ -23,32 +23,32 @@ bool presetInfoNameComparator(const PresetInfo &a, const PresetInfo &b) {
 }
 } // namespace
 
-PresetInfoEnumerator::PresetInfoEnumerator(const QString& searchPath)
-        : PresetInfoEnumerator(QList<QString>{searchPath}) {
+MappingInfoEnumerator::MappingInfoEnumerator(const QString& searchPath)
+        : MappingInfoEnumerator(QList<QString>{searchPath}) {
 }
 
-PresetInfoEnumerator::PresetInfoEnumerator(const QStringList& searchPaths)
+MappingInfoEnumerator::MappingInfoEnumerator(const QStringList& searchPaths)
         : m_controllerDirPaths(searchPaths) {
-    loadSupportedPresets();
+    loadSupportedMappings();
 }
 
-QList<PresetInfo> PresetInfoEnumerator::getPresetsByExtension(const QString& extension) {
+QList<MappingInfo> MappingInfoEnumerator::getMappingsByExtension(const QString& extension) {
     if (extension == MIDI_PRESET_EXTENSION) {
```

Comment:
Do we want to keep the `_PRESET_EXTENSION` names or rename them to `_MAPPING_EXTENSION` for consistency